app/gzac/src/main/java/com/ritense/gzac/listener/ApplicationReadyEventListener.kt

```diff
@@ -499,7 +499,7 @@ class ApplicationReadyEventListener(
                 properties = jacksonObjectMapper().readValue(
                     """
                     {
-                        "url": "http://host.docker.internal:8011/api/v1/",
+                        "url": "http://localhost:8011/api/v1/",
```

Comment:
Please don't change this

Reply:
I've changed it to make it work on localhost. I also added [host.docker.internal](https://github.com/valtimo-platform/valtimo-backend-libraries/pull/551/files#diff-e8e91b8487bd96a2d46f1eab1619d7e9644d89cb750ee3fc71d8a73fed7ec1edR53) in other places, for the same reason